src/sentry/event_manager.py

```diff
@@ -2418,19 +2418,37 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects: Projec
 
 @metrics.wraps("performance.performance_issue.should_create_group", sample_rate=1.0)
 def should_create_group(client: Any, grouphash: str, type: GroupType) -> bool:
-    times_seen = client.incr(f"grouphash:{grouphash}")
-    metrics.incr(
-        "performance.performance_issue.grouphash_counted",
-        tags={"times_seen": times_seen, "group_type": GROUP_TYPE_TO_TEXT.get(type, "Unknown Type")},
-        sample_rate=1.0,
-    )
+    with sentry_sdk.start_span(op="event_manager.should_create_group") as span:
```

Comment:
just for clarity, op can be `event_manager.should_create_perf_group`

Reply:
I'm just mirroring the name of the function that this span wraps, should I rename the function too?

Reply:
oh I see. hm, ok yeah no need to rename it then 👍